README.md

```diff
@@ -1 +1,102 @@
-# deployment-deploy-services
\ No newline at end of file
+# How to deploy dalite-ng on OpenStack server
+ 
+This repository contains ansible deployment scripts allowing one to deploy dalite-ng to OpenStack server.
+
+
+Create a OpenStack environment
+------------------------------
+
+1. Create a root volume that has 40GB size. It should be based on Ubuntu 14.04 image. 
+   We suggest you use vanilla image from [ubuntu page](https://cloud-images.ubuntu.com/). 
```

Comment:
@jbzdak I'm not sure that the changes in 7efa934 explain how to use the vanilla ubuntu image. Is the ubuntu image in the image list the vanilla image or OVH's image? I think you have to upload the image first using the openstack cli:

    glance image-create --disk-format=qcow2 --container-format=bare --file <img> --name <name> --progress

@antoviaque can you confirm?